packages/config/config/devices/0x0455/ora-zrx.json

```diff
@@ -0,0 +1,203 @@
+{
+	"manufacturer": "Cherubini S.p.A.",
+	"manufacturerId": "0x0455",
+	"label": "Ora ZRX",
+	"description": "Z-Wave Tubular Motor",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Lifeline group",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Follow Me",
+			"description": "The device in this group level",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Scene Activation",
+			"description": "Receive an activation scene ID if an obstacle are reach during its operation. The scene ID can be define by using parameters 30, 31",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Follow Me Weather",
+			"description": "The device in this group will follow the device level if the change is due by the weather condition. The parameter ID sent by this command can be configured by using parameter 32",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "30",
+			"label": "Open Collision Scene",
+			"description": "Scene ID sent if a collision is detected during opening operation",
```

Comment:
As a rule, we try to minimize having descriptions unless they actually add anything. Let's try using "Send Scene ID: Jam When Opening" and removing the description.

Reply:
Thanks, understood